whedon.rb

```diff
@@ -397,9 +397,8 @@ def create_or_update_git_branch(issue_id, papers_repo, journal_alias)
     pdf_path = "#{journal_alias}.#{id}/10.21105.#{journal_alias}.#{id}.pdf"
 
     begin
-      # If the PDF is there already then delete it
-      github_client.contents(papers_repo, :path => pdf_path, :ref => "heads/#{journal_alias}.#{id}")
-      blob_sha = github_client.contents(papers_repo, :path => pdf_path, :ref => "heads/#{journal_alias}.#{id}").sha
+      ref_sha = github_client.refs(papers_repo, "heads/#{journal_alias}.#{id}").object.sha
```

Comment:
- Check if there is a ref at e.g. `heads/joss.00491`, if there is, then look up the commit and get the sha of the first file (which will be the previously compiled PDF).
- Then delete the old file.